src/sentry/nodestore/bigtable/backend.py

```diff
@@ -305,6 +39,11 @@ def __init__(
         compression=False,
         **client_options,
     ):
+        if compression is True:
+            compression = "zlib"
+        elif compression is False:
+            compression = None
```

Comment:
`BigtableKVStorage.compression` was changed to `Optional[str]`, so this changes the previous boolean parameters into their equivalents so that future users of `BigtableKVStorage` don't have to think about it.